examples/gno.land/p/demo/users/types.gno

```diff
@@ -12,3 +16,14 @@ func (aon AddressOrName) GetName() (string, bool) {
 	}
 	return "", false
 }
+
+func (aon AddressOrName) IsValid() bool {
```

Comment:
what about using `users.Resolve(aon).IsValid()`?

Reply:
can't call the realm method `users.Resolve` from a `p` package